Header header
bool power_cord
bool dock
float32 level			# in %
uint32 time_remaining		# in minutes

